core/domain/rights_manager.py

```diff
@@ -628,6 +628,16 @@ def _publish_activity(committer_id, activity_id, activity_type):
         committer_id, activity_id, activity_type, ACTIVITY_STATUS_PUBLIC,
         '%s published.' % activity_type)
 
+    #Add contribution dates to the owners and editors
+    exploration_rights = get_exploration_rights(activity_id)
```

Comment:
activity_rights; s/exploration/activity generally if you're doing the update in this method